<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleObservation</class>
 <widget class="QWidget" name="SingleObservation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Single Observation</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons:singleobservation.svg</normaloff>icons:singleobservation.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,1">
   <property name="margin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="labelStation">
     <property name="text">
      <string>Station:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0,0,0,0,0,0,0,1">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLineEdit" name="textStation">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelLevel">
       <property name="text">
        <string>L:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="textLevel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Level</string>
       </property>
       <property name="maxLength">
        <number>5</number>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSensorNr">
       <property name="text">
        <string>S:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="textSensorNr">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Sensor Number</string>
       </property>
       <property name="maxLength">
        <number>5</number>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelType">
       <property name="text">
        <string>Type:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="textType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="maxLength">
        <number>5</number>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelParam">
       <property name="text">
        <string>Param:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="textParam">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="maxLength">
        <number>5</number>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelObstime">
       <property name="text">
        <string>Obstime:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="textObstime">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelOriginal">
     <property name="text">
      <string>Orig:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="textOriginal">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="buttonAcceptOriginal">
     <property name="toolTip">
      <string>Accept the original value</string>
     </property>
     <property name="text">
      <string>Acc</string>
     </property>
     <property name="icon">
      <iconset>icons:accept.svg</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+G</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4" rowspan="6">
    <widget class="QTabWidget" name="tabs">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabChecks">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Checks</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_18" stretch="1">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSmallTable" name="tableChecks">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>20</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabHistory">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>History</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_17">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableView" name="tableHistory"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelModel">
     <property name="text">
      <string>Model:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="textModel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="buttonAcceptModel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Accept the model value</string>
     </property>
     <property name="text">
      <string>Acc</string>
     </property>
     <property name="icon">
      <iconset>icons:accept.svg</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+H</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelFlags">
     <property name="text">
      <string>Flags:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="textFlags">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelCorrected">
     <property name="text">
      <string>&amp;Corr:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>comboCorrected</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="comboCorrected">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QToolButton" name="buttonEditor">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Start an editor for this observation</string>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
     <property name="icon">
      <iconset>icons:editor.svg</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+E</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QToolButton" name="buttonAcceptCorrected">
     <property name="toolTip">
      <string>Accept the corrected value</string>
     </property>
     <property name="text">
      <string>Acc</string>
     </property>
     <property name="icon">
      <iconset>icons:accept.svg</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+J</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="checkQC2">
     <property name="toolTip">
      <string>Allow QC2 to override</string>
     </property>
     <property name="text">
      <string>QC&amp;2</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QToolButton" name="buttonReject">
     <property name="toolTip">
      <string>Reject the observation</string>
     </property>
     <property name="text">
      <string>Rej</string>
     </property>
     <property name="icon">
      <iconset>icons:reject.svg</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+K</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>69</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>QSmallTable</class>
   <extends>QTableView</extends>
   <header>util/QSmallTable.hh</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboCorrected</tabstop>
  <tabstop>buttonEditor</tabstop>
  <tabstop>buttonAcceptOriginal</tabstop>
  <tabstop>buttonAcceptModel</tabstop>
  <tabstop>buttonAcceptCorrected</tabstop>
  <tabstop>buttonReject</tabstop>
  <tabstop>checkQC2</tabstop>
  <tabstop>tabs</tabstop>
  <tabstop>tableChecks</tabstop>
  <tabstop>tableHistory</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonAcceptOriginal</sender>
   <signal>clicked()</signal>
   <receiver>SingleObservation</receiver>
   <slot>onAcceptOriginal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAcceptModel</sender>
   <signal>clicked()</signal>
   <receiver>SingleObservation</receiver>
   <slot>onAcceptModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAcceptCorrected</sender>
   <signal>clicked()</signal>
   <receiver>SingleObservation</receiver>
   <slot>onAcceptCorrected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonReject</sender>
   <signal>clicked()</signal>
   <receiver>SingleObservation</receiver>
   <slot>onReject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonEditor</sender>
   <signal>clicked()</signal>
   <receiver>SingleObservation</receiver>
   <slot>onStartEditor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboCorrected</sender>
   <signal>activated(QString)</signal>
   <receiver>SingleObservation</receiver>
   <slot>onNewCorrected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkQC2</sender>
   <signal>toggled(bool)</signal>
   <receiver>SingleObservation</receiver>
   <slot>onQc2Toggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAcceptOriginal()</slot>
  <slot>onAcceptModel()</slot>
  <slot>onAcceptCorrected()</slot>
  <slot>onStartEditor()</slot>
  <slot>onNewCorrected()</slot>
  <slot>onReject()</slot>
  <slot>onQc2Toggled(bool)</slot>
 </slots>
</ui>
